### PR TITLE
Feat/ade 495/create new index page

### PR DESCRIPTION
**Pull request by SamMurphyDev**

### CODE REVIEW

**Review by robinMcA — COMMENTED** [review]
Looks good to me

File: src/main/java/au/com/agiledigital/idea_search/macros/TemplateId.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package au.com.agiledigital.idea_search.macros;
+
+import com.atlassian.confluence.content.render.xhtml.ConversionContext;
+import com.atlassian.confluence.macro.Macro;
+import com.atlassian.confluence.macro.MacroExecutionException;
+import java.util.Map;
+
+/**
+ * Container macro for the
```
[the remark this answers is not in the document]
Comment (SamMurphyDev) [reply src/main/java/au/com/agiledigital/idea_search/macros/TemplateId.java]: Finished the sentence

File: src/main/java/au/com/agiledigital/idea_search/macros/IndexTable.java [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.w3c.dom.ls.DOMImplementationLS;
+import org.w3c.dom.ls.LSSerializer;
+import org.xml.sax.SAXException;
+
+/**
+ * Macro for the Index Table. Fetches the pages with the label "fedex-ideas", pulls the structured
```
[the remark this answers is not in the document]
Comment (SamMurphyDev) [reply src/main/java/au/com/agiledigital/idea_search/macros/IndexTable.java]: Added additional context

**Review by robinMcA — APPROVED** [review]

**Pull request merged**